frontend/src/pages/spotnet/dashboard/Dashboard.jsx

```diff
@@ -218,15 +218,15 @@ export default function Component({ walletId, telegramId }) {
               </div>
             )}
           </div>
-
-          <button className="redeem-button" onClick={() => closePositionEvent()} disabled={isClosing}>
+          <Button variant="primary" size="lg" className="" onClick={() => closePositionEvent()} disabled={isClosing}>
```

Comment:
it is broken on mobile (please take a look into mobile's Figma), you should use smaller size for mobile devices

![image](https://github.com/user-attachments/assets/194c137f-2073-4431-a8d1-92c0572a480f)

Maybe the best way for our case is to handle it in css:
so, looking into figma it seems like we can have such cases:
lg is 609px for desktop and 324px for mobile 
md is 309px for desktop and 167px for mobile

Reply:
The issue with this part layout of the dashboard, it is not responsive on mobile screen, hence the overlapping, but i have fixed it, will be making another push